pkg/restore/restore.go

```diff
@@ -458,92 +454,51 @@ func (ctx *restoreContext) execute() (Result, Result) {
 	// totalItems: previously discovered items, i: iteration counter.
 	totalItems, i, existingNamespaces := 0, 0, sets.NewString()
 
-	for _, selectedResource := range selectedResourceCollection {
+	// First restore only CRDs
```

Comment:
I think it would be useful to expand this comment to state why CRDs are getting special treatment here. Even just adding something like "... so that they are available in the cluster for validation of custom resource objects" would be enough.